packages/story-editor/src/components/footer/carousel/carouselLayout.js

```diff
@@ -54,24 +98,53 @@ const Area = styled.div`
 `;
 
 function CarouselLayout() {
-  const { numPages } = useCarousel(({ state: { numPages } }) => ({ numPages }));
+  const { carouselState } = useLayout(({ state: { carouselState } }) => ({
+    carouselState,
+  }));
+
+  const { numPages, pageThumbHeight } = useCarousel(
+    ({ state: { numPages, pageThumbHeight } }) => ({
+      numPages,
+      pageThumbHeight,
+    })
+  );
 
   if (numPages <= 0) {
     return null;
   }
 
+  const isCollapsed = carouselState === CAROUSEL_STATE.CLOSED;
+
+  const isOpenOrOpening = [
+    CAROUSEL_STATE.OPEN,
+    CAROUSEL_STATE.OPENING,
+  ].includes(carouselState);
+
   return (
-    <Wrapper aria-label={__('Page Carousel', 'web-stories')}>
-      <Area area="prev-navigation">
-        <CarouselScrollBack />
-      </Area>
-      <Area area="carousel">
-        <CarouselList />
-      </Area>
-      <Area area="next-navigation">
-        <CarouselScrollForward />
-      </Area>
-    </Wrapper>
+    <CSSTransition in={isOpenOrOpening} classNames="carousel" timeout={700}>
```

Comment:
Use the constant instead of 700 here?

Reply:
Done in 078eda9b34. Also increased the speed to 300ms.